networking/determine-best-networking.md

```diff
@@ -1,110 +1,219 @@
+
+**{{site.prodname}} CNI IPAM plugin**
+
+The {{site.prodname}} CNI IPAM plugin allocates IP addresses for pods out of one or more configurable IP address ranges, dynamically allocating small blocks of IPs per node as required.  The result is a more efficient IP address space usage compared to many other CNI IPAM plugins, including the host local IPAM plugin which is used in many networking solutions.
+
+**Overlay network modes**
+
+{{site.prodname}} can provide both VXLAN or IP-in-IP overlay networks, including cross-subnet only modes.
+
+**Non-overlay network modes**
+
+{{site.prodname}} can provide non-overlay networks running on top of any underlying L2 network, or an L3 network that is either a public cloud network with appropriate cloud provider integration, or a BGP capable network (typically an on-prem network with standard Top-of-Rack routers).
+
+**Network policy enforcement**
+
+{{site.prodname}}’s network policy enforcement engine implements the full range of Kubernetes Network Policy features, plus the extended features of {{site.prodname}} Network Policy.  This works in conjunction with {{site.prodname}}’s built in networking modes, or any other {{site.prodname}} compatible network plugins and cloud provider integrations.
+
+### {{site.prodname}} compatible CNI plugins and cloud provider integrations
+In addition to the {{site.prodname}} CNI plugins and built in networking modes, {{site.prodname}} is also compatible with a number of third party CNI plugins and cloud provider integrations.
+
+**Amazon VPC CNI**
+
+The Amazon VPC CNI plugin allocates pod IPs from the underlying AWS VPC and uses AWS elastic network interfaces to provide VPC native pod networking (pod IPs that are routable outside of the cluster). It is the default networking used in {% include open-new-window.html text='Amazon EKS' url='https://aws.amazon.com/eks/' %}, with Calico for network policy enforcement.
 
 **Azure CNI**
 
-The Azure CNI plugin configures the Azure virtual network to provide pod networking. It is the default networking used in Microsoft AKS, with {{site.prodname}} for [network policy enforcement]({{ site.baseurl }}/security/calico-network-policy).
+The Azure CNI plugin allocates pod IPs from the underlying Azure VNET configures the Azure virtual network to provide VNET native pod networking (pod IPs that are routable outside of the cluster). It is the default networking used in {% include open-new-window.html text='Microsoft AKS' url='https://azure.microsoft.com/en-us/services/kubernetes-service/' %}, with Calico for network policy enforcement.
+
+**Google cloud provider**
+
+The Google cloud provider integration uses host-local IPAM CNI plugin to allocate pod IPs and programs the Google cloud network Alias IP ranges to provide VPC native pod networking on Google cloud (pod IPs that are routable outside of the cluster). It is the default for Google Kubernetes Engine (GKE), with Calico for network policy enforcement.
```

Comment:
The Google cloud provider integration uses host-local IPAM CNI plugin to allocate pod IPs, (comma or it reads "allocate IPs and programs")

{{site.prodname}}

Reply:
I don't think this shouldn't be site.prodname.  GKE is integrated with Calico, not Calico Enterprise.  So this stays Calico even in the enterprise docs.